python/tank/descriptor/io_descriptor/shotgun_entity.py

```diff
@@ -316,67 +308,21 @@ def get_latest_version(self, constraint_pattern=None):
     def get_latest_cached_version(self, constraint_pattern=None):
         """
         Returns a descriptor object that represents the latest version
-        that is locally avaiable in the bundle cache search path.
+        that is locally available in the bundle cache search path.
 
-        :param constraint_pattern: If this is specified, the query will be constrained
-               by the given pattern. Version patterns are on the following forms:
-
-                - v0.1.2, v0.12.3.2, v0.1.3beta - a specific version
-                - v0.12.x - get the highest v0.12 version
-                - v1.x.x - get the highest v1 version
+        :param constraint_pattern: This parameter is unused.
```

Comment:
nitpick: suggest slight tweak to param doc here. Something like:

`This parameter is unused and remains here for backward compatibility with a legacy implementation`.